src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/lecture-unit-management.component.ts

```diff
@@ -186,6 +193,17 @@ export class LectureUnitManagementComponent implements OnInit, OnDestroy {
         });
     }
 
+    isViewButtonAvailable(lectureUnit: LectureUnit) {
```

Comment:
Can you add the return type?
```suggestion
    isViewButtonAvailable(lectureUnit: LectureUnit): boolean {
```

Reply:
Added